### PR TITLE
Update files with PEP8 standards in folder dojo #001

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/

Folder affected
dojo/tools
dojo/user
dojo/tool_type
dojo/tool_product
dojo/tool_config
dojo/test_type
dojo/test
dojo/templatetags
dojo/templates
dojo/system_settings
dojo/survey
dojo/sla_config
dojo/settings
dojo/search
dojo/risk_acceptance
dojo/request_cache
dojo/reports
dojo/regulations

### CODE REVIEW

File: dojo/settings/unittest.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 # first;
 # Do so by copying the file dojo/settings/settings.dist.py to
 # dojo/settings/settings.py; Otherwise, the following import will not work
-from .settings import *
+# from .settings import *
```
Comment (Maffooch) [inline dojo/settings/unittest.py:5]: I know this is likely redundant, but is probably worth keeping

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/settings/unittest.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/settings/unittest.py]: Uncomment line in this commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/586e450a15dbd850adccd50b095b306374bcdecf

File: dojo/templatetags/event_tags.py [hunk 2]
```diff
@@ -79,10 +81,10 @@ def sum_dict(d):
...
 @register.filter
 def nice_title(title):
-    pat = re.compile(r'Finding [0-9][0-9][0-9]:*')
+    pat = re.compile(r"Finding [0-9][0-9][0-9]:*")
     s = pat.split(title, 2)
     try:
         ret = s[1]
         return ret
-    except:
+    except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/templatetags/event_tags.py]: Changes apply in this commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/586e450a15dbd850adccd50b095b306374bcdecf

File: dojo/reports/widgets.py [hunk 3]
```diff
@@ -26,26 +26,38 @@ class CustomReportJsonForm(forms.Form):
     json = forms.CharField()
 
     def clean_json(self):
-        jdata = self.cleaned_data['json']
+        jdata = self.cleaned_data["json"]
         try:
-            json_data = json.loads(jdata)
-        except:
+            json.loads(jdata)
+        except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/reports/widgets.py]: Changes apply in this commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/586e450a15dbd850adccd50b095b306374bcdecf

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/settings/unittest.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/settings/unittest.py:5]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/cf26b9343174595697195bd93067a56a13fc9bda

**Review by Maffooch — APPROVED** [review]

**Comment by Maffooch** [comment]
Looks like fix conflicts reintroduced a lot of non pep8 compliant code

**Comment by Maffooch** [comment]
Closing due to stale. If this is an error, please reopen

**Pull request closed without merging**